ydb/core/base/blobstorage.h

```diff
@@ -481,6 +481,7 @@ struct TEvBlobStorage {
         // user <-> proxy interface
         EvPut = EventSpaceBegin(TKikimrEvents::ES_BLOBSTORAGE), /// 268 632 064
         EvGet,
+        EvGetBlock,
```

Comment:
always add new events to the end of block -- they potentially may be transferred over network